activiti-cloud-services-org/activiti-cloud-services-org-rest/pom.xml

```diff
@@ -129,5 +128,10 @@
       <artifactId>spring-cloud-starter-contract-verifier</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.activiti.cloud</groupId>
+      <artifactId>activiti-cloud-services-org-core</artifactId>
+      <version>${version}</version>
```

Comment:
version is set by parent and can be removed here

Reply:
done